lib/git_blame.rb

```diff
@@ -26,7 +26,7 @@ def changes_since_last_green(build)
     def changes_in_branch(build)
       output = GitRepo.inside_repo(build.repository) do
         # TODO: Push this down into GitRepo and integration test it.
-        Cocaine::CommandLine.new("git log --cc --format='::!::%H|%cn <%ce>|%cd|%B::!::' 'origin/master..origin/#{build.branch_record.name}'").run
+        Cocaine::CommandLine.new("git log --cc --format='::!::%H|%cn <%ce>|%cd|%B::!::' 'master..#{build.branch_record.name}'").run
```

Comment:
Since we're anyway modifying these lines, maybe write a helper routine to generate the command-line? there's some copy paste between here and the next line